strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/testing/repository/MavenRepositoryManagementTest.java

```diff
@@ -0,0 +1,59 @@
+package org.carlspring.strongbox.testing.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.carlspring.strongbox.config.Maven2LayoutProviderTestConfig;
+import org.carlspring.strongbox.providers.layout.Maven2LayoutProvider;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.testing.storage.repository.RepositoryManagementTestExecutionListener;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+
+/**
+ * @author ankit.tomar
+ */
+@SpringBootTest
+@ActiveProfiles(profiles = "test")
+@ContextConfiguration(classes = Maven2LayoutProviderTestConfig.class)
+@Execution(ExecutionMode.CONCURRENT)
+public class MavenRepositoryManagementTest
+{
+
+    private static final String MAVEN_STORAGE = "maven-storage-to-be-injected";
```

Comment:
Our convention is to use the initials of the test's name. In this case, it should be `mrmt-storage` and `mrmt-releases`.